### DISCUSSION TITLE
Initialize another colang content from the Main colang content

### DISCUSSION THREAD

**Discussion by Sudhu2004** [NVIDIA-NeMo/Guardrails#195] [Q&A]
Hi team! I was wondering how do I use the "import" and "include" keywords in the Main colang content so execute flows from other colang content I have defined!

I got to know about this "import" and "include" over  [here](https://github.com/NVIDIA/NeMo-Guardrails/blob/main/docs/user_guide/colang-syntax-reference.md)

can you please say me how to use those keywords in the colang content

**Comment by xywen97** [comment]
The guardrail may do the combination of multiple colang files automatically. But I'm also curious about the usage of the goto and label keywords, can you use them?

**Comment by Sudhu2004** [comment]
I am still figuring out how to use them....
@drazvan Can you please provide the solution for this problem we are facing

**Comment by drazvan** [comment]
Hi @Sudhu2004!

While from a syntactic point of view, Colang supports the "include/import" statements, they are not yet active from a functional perspective. As @xywen97 said, currently, all the Colang files are merged together. But this will be added in a future version. 

@xywen97 : Yes, the `label` and `goto` should work, although they are only meant for simple loops (otherwise the flow can get very difficult to follow). 

```
define flow 
   ...
   label x 
   ...
   goto x
```
